packages/atomic/src/components/facets/facet-segmented-value/facet-segmented-value.pcss

```diff
@@ -0,0 +1,20 @@
+.box-container li:first-child .value-box {
+  @apply rounded-lr;
+}
+
+.box-container li:last-child .value-box {
+  @apply rounded-rr;
+}
+
+.value-box .value-label {
+  @apply group-hover:text-primary-light;
+  max-width: 15ch;
+}
+
+.box-container .value-box.selected {
+  @apply border-primary shadow-inner-primary;
```

Comment:
Wasn't able to apply these classes directly on the component as for some reason the border-primary class would not be applied and the shadow-inner-primary class would not apply the "inset" option inside of "inset 0 0 0 1px var(--atomic-primary)"